src/opera/commands/package.py

```diff
@@ -0,0 +1,68 @@
+import argparse
+import uuid
+
+from pathlib import Path
+
+from opera.error import DataError, ParseError
+from opera.parser.tosca.csar import CloudServiceArchive
+
+
+def add_parser(subparsers):
+    parser = subparsers.add_parser(
+        "package",
+        help="Package service template and all accompanying files into a CSAR"
+    )
```

Comment:
Needs the verbose switch to be consistent with other commands.

Reply:
Dunzo.